server/controllers/lettersControllers/lettersControllers.js

```diff
@@ -68,8 +104,19 @@ exports.updateLetters = async (req, res) => {
 
 exports.deleteById = async (req, res) => {
     try {
+        const {id} = req.params
+        const {quantity} = req.body
 
-    } catch (e) {
+        const letter = await Letters.findByIdAndDelete(id)
```

Comment:
This is actually deleting an entire Letters document. We can remove the `quantity` variable. Anything changing with `quantity` will be done in the update controller.

Reply:
So, for this, I'm confused as to what we would be deleting if we are not deleting a Letters document

Reply:
As in, if we are using update to update the quantity, price, etc, what do we need the Delete function for?

Reply:
Let's assume that a certain font, type, and size of letter is no longer available, they discontinued it, or otherwise have no desire to list it anymore... would we want to delete it? or keep it in the db? It's simply an option for something we may or may not use.

Reply:
So this would simply be something where we can say "isAvailable=false" instead of just deleting the document itself form the db?

Reply:
We could do that... absolutely assuming that it will be returning like a special or something. We would just have to put an extra field on the schema and set it as a boolean. Then add a default as true because if we are creating it, we can assume it's available. We can get rid of the delete functionality all together and filter in get all for isAvailable and update in in the put method. If that's what you'd prefer.